melosys-eessi-app/src/main/java/no/nav/melosys/eessi/controller/interceptor/CorrelationIdInterceptor.java

```diff
@@ -0,0 +1,46 @@
+package no.nav.melosys.eessi.controller.interceptor;
+
+import java.util.Objects;
+import java.util.UUID;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+import org.springframework.web.servlet.HandlerInterceptor;
+
+import static no.nav.melosys.eessi.config.MDCLogging.CORRELATION_ID;
+import static no.nav.melosys.eessi.config.MDCLogging.X_CORRELATION_ID;
```

Comment:
Når jeg tenker meg om, så har vel ikke X_CORRELATION_ID noe med MDC Logging å gjøre? Det er vel strengt talt for header

Reply:
Det er en standard header ja, men jeg har samlet alt ang correlation i MDCLogging